.github/CONTRIBUTING.md

```diff
@@ -154,15 +154,158 @@ the repo such that they can get coding -->
+
+Consider the following issue: 
+> Hovering a `kirby-button` with attention level 2 in IE11 does not activate the hover-effect. It works when i use attention level 1, but i get warnings in the console.
+
+If that issue is closed, how is the reader supposed to know what was actually fixed? 
+Was it the issue of the hover-effect not activating? The issue of warnings being thrown when using attention level 1? Both? 
+
+To solve this, the issue should be split into two seperate issues which refer to each other: 
+
+Issue #1:
+> Hovering a `kirby-button` with attention level 2 in IE11 does not activate the hover-effect. This might be related to issue #2.
+
+Issue #2:
+> I am not able to use a `kirby-button` with attention level 1 without a warning being thrown in the console. This might be related to issue #1.
+
+If you feel like you are "complaining" too much by creating several issues; don't! 
+It is only helpful of you to split up your issues - you are saving time and easing organization. 
+
+#### The good issue is self-contained 
+<!-- - Self-contained
     - Screenshots if necessary
-    <!-- I think the following point is important if Kirby is to be spread out amongst several teams -->
+     I think the following point is important if Kirby is to be spread out amongst several teams
     - Don't assume people have knowledge outside of Kirby - contributors might come from many different projects 
+- Explain what you want ad why you want it. -->
+It can not be assumed that the reader of your issue has any knowledge of your work outside of Kirby. 
+
+The reader might be: 
+- a contributor outside of bank data 
+- someone who is not in your team 
+- potentially a new hire
+
+This also goes for Kirby maintainers - some of which has been hired to specifically work on Kirby. 
+
+Therefore: 
+- Do not use internal lingo 
+- Do not assume the reader can access the application you are working on - include screenshots instead.
+- Do not assume the reader can access the repository you are working in - include code snippets instead (if possible). 
+
+<!--- Be specific and unambigious -->
+#### The good issue is specific & unambigious
+<!--- Do point to relevant files if you know them, don't just say "there's some css files". 
+- Facts first opinions second. -->
+
+The wording of an issue should be as specific and unambigious as possible. This will help the reader understand exactly what you mean. This can be done in many ways. Therefore we have compiled some examples to demonstrate:
+
+_Example #1: referring to files_ 
+
+If you are referring to a file in the repository do not refer to it as `_typography.css` use the full path instead: `libs/designsystem/src/lib/scss/base/_typography.scss`.
+While it might be obvious for you and experienced contributors, what and where `_typography.css` is; it is not obvious for newcomers. 
+
+If you already have refered to the full path earlier in the issue, it can be okay to refer to the file as `_typography.css`.
```

Comment:
New section (I've spaced _# 1_ on purpose just for this comment, otherwise it would link to an issue 😄): 

> ##### Example # 1: Referring to files 
>
> If you are referring to a file in the repository do not refer to it as `_typography.css` use the full path instead: `libs/designsystem/src/lib/scss/base/_typography.scss`.
>
> It can also be helpful to directly link to the file such that the reader easily can access it: [`libs/designsystem/src/lib/scss/base/_typography`](/kirbydesign/designsystem/blob/master/libs/designsystem/src/lib/scss/base/_typography.scss).                          
>
>While it might be obvious for you and experienced contributors, what and where `_typography.css` is; it is not obvious for newcomers.
>
> If you already have refered to the full path earlier in the issue, it is okay to refer to the file as `_typography.css` without linking to it.

Reply:
Nice. At least you would have linked to the very first issue 😁.

About _"linkt to the file_  comment. I may have thought of a permalink to the file in the repository:

https://github.com/kirbydesign/designsystem/blob/96bdb1d20c1e0cd58767b03e00515dee9af81e0d/libs/designsystem/src/lib/scss/base/_typography.scss

Maybe combined with the path as link text:

[libs/designsystem/src/lib/scss/base/_typography.scss](https://github.com/kirbydesign/designsystem/blob/96bdb1d20c1e0cd58767b03e00515dee9af81e0d/libs/designsystem/src/lib/scss/base/_typography.scss)